### ISSUE TITLE
v6 campaing balance is EXECRABLE

### ISSUE THREAD

**Issue by mwoehlke** [Anuken/Mindustry-Suggestions#1091]
**Describe what you would like changed, and why.**
Make the campaign actually possible to win. Only "ground zero" is beatable; the waves in the other levels end up being able to overrun any defenses that can be built with the available resources by around the halfway mark of the number of waves that must be defeated. By about wave 25, it is impossible to even rebuild defenses faster than waves spawn and wipe them out.

**Describe the changes you want to propose. Include possible alternatives.**
- Remove ability for enemies — at least low level enemies ­— to shoot through walls. (Also, enemies should not be able to shoot through surge walls at all; I don't know if they can, and I expect I'll never know without inspecting the source, since there is exactly zero possibility of getting to that point in the campaign.)
- Don't introduce enemies significantly above the player's tech level. Arc enemies in difficulty-2 missions, when the player doesn't even have graphite yet and can only fight back with duos, are severely unbalanced. Lancer enemies when the player has *just* unlocked silicon are ***light years beyond*** "unbalanced".
- Nerf lancer enemies or make them only show up around surge tier; an enemy that can annihilate huge swaths of defenses in a couple seconds is a great way to lose players permanently.
- Don't exponentially increase the difficulty after a failed mission as this makes the game unwinnable.

**Comment by DayTimer** [comment]
Do not play difficulty 6 sectors untill you research a ripple and a cyclone. I've captured a lot of sectors already so I can say with confidence that it is beatable.

Capture g0, then capture craters and farm graphite here. You can do it using only duos. Research hail, titanium and a pneumo drill, launch to a sector with some titanium in it. Research ripple. Further it will be very easy.

**Comment by DeltaNedas** [comment]
git gud skrub

**Comment by potatoisagender** [comment]
^
Also you aren't supposed to capture neighbouring tiles. Do all the researched ones first...

**Comment by Sanya2907** [comment]
Campaign is perfect at the moment, except guardians that are too ducking overpowered

**Comment by BlueWolf3682** [comment]
capture 175, dont touch frozen forest unless youre ready to attack an enemy base or lose it

**Comment by mwoehlke** [comment]
> Do not play difficulty 6 sectors

This was a difficulty *two* sector ("174"), and one that doesn't even contain titanium:
![image](https://user-images.githubusercontent.com/3236798/98450184-ad8bb000-2108-11eb-879c-fa2ae06fd13e.png)

After resetting my campaign to get out from being absolutely pwned, I've since played different difficulty-2 sectors, and yeah, they're okay. (Mostly¹.) I'm thinking 174 is just plain broken. That sector simply doesn't *have* the resources to survive 40 waves.

(¹ Being *guaranteed* to have to rebuild something after only 4 bombers is incredibly irritating. At least it's usually only a couple conveyor sections, but still, having to constantly fly over there to find what got destroyed makes it hard to work on infrastructure. And where are my repair drones from v4?)

**Comment by DayTimer** [comment]
> And where are my repair drones from v4?
Poly automatically rebuilds destroyed buildings. Mega automatically repairs damaged buildings.

**Comment by mwoehlke** [comment]
> Mega automatically repairs damaged buildings.

I don't have that yet. Why is a unit that can *rebuild* buildings lower on the tech tree than one that can *repair*? In v4, repair drones only need silicon and metaglass (and lead, but that's a *starting* resource). I want those back.

**Comment by matkoniecz** [comment]
> Why is a unit that can rebuild buildings lower on the tech tree than one that can repair?

Why not?

> I want those back.

You can play v4. Or insult v6.

> Only "ground zero" is beatable

Get better in this game rather than claim that everyone in as poor in playing it as you.

Hint: overly reaching and overinflated claims are not going to be treated seriously.

**Comment by mwoehlke** [comment]
> > Only "ground zero" is beatable
>
> Get better in this game rather than claim that everyone in as poor in playing it as you.

Or *you* could read the following comments. It turns out the problem is 174, specifically. I just made the mistake of attempting that sector second, which was an unmitigated disaster. Other sectors were much easier. I stand by my assertion that getting spammed by Lancer enemies in a sector with no titanium is unbalanced. I'd dearly like to see how *you* defend against those when your tech tree doesn't even include silicon.

Expecting everyone is as god-like as you is equally unfair.

>> Why is a unit that can rebuild buildings lower on the tech tree than one that can repair?
>
> Why not?

...because rebuilding is *much* more useful than merely repairing. Plus, Poly can *already* repair, just really badly because it requires Logic and has to be toggled manually if you want them to also be able to rebuild. Which is a PITA. And something I fixed locally, which has made the game *much* more enjoyable. (Now, if only ye olde 'pew' didn't make the audio pop for reasons unknown...)

> You (and similar people) are reason for https://github.com/Anuken/Mindustry-Suggestions#i-will-no-longer-be-commenting-or-closing-issues-here

*You* (and similar people) are why the atmosphere around here is so toxic.
